server/js/sockets.js

```diff
@@ -288,21 +287,19 @@ sockets.init = function socketInit(server) {
           getPropWithPath(retained.status, path),
         );
     });
-
+    // for wireless-module
     socket.on('get-payload', (path) => {
       if (path instanceof Array && path.length > 0)
         socket.emit(path.join('-'), getPropWithPath(retained, path));
     });
```

Comment:
Oh it was just for me sorry because I was about to make a function so that the retained data in the server were kept on reload, but then I realised that that function basically does that so I can remove that.